### PR TITLE
hy/ Test open bookmarks from toolbar

### PR DESCRIPTION
**Pull request by sv-hyacoub**
# Description

Verify that the user can open Bookmarks from the Toolbar with a mouse click

## Bugzilla bug ID

**[Testrail](https://mozilla.testrail.io/index.php?/cases/view/2084550)**
**[Link](https://bugzilla.mozilla.org/show_bug.cgi?id=1910732)**

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

Completed

# Screenshots / Explanations

N/A

# Comments / Concerns

N/A

### CODE REVIEW

**Review by ben-c-at-moz — DISMISSED** [review]
Looks good, I have a suggestion but even if you don't want to implement it, this is a good test

File: tests/bookmarks_and_history/test_open_bookmarks_from_toolbar.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    with driver.context(driver.CONTEXT_CHROME):
+        nav.get_element("star-button").click()
+        nav.get_element("save-bookmark-button").click()
+
+        # Open new tab and click on the bookmark from the Bookmarks Toolbar
+        newtab.new_tab_by_button()
+        panel.get_element("bookmark-by-title", labels=["Internet for people"]).click()
+
+    # Verify that the page is loaded
+    WebDriverWait(driver, 10).until(EC.title_is("Internet for people, not profit — Mozilla (US)"))
```
Comment (ben-c-at-moz) [inline tests/bookmarks_and_history/test_open_bookmarks_from_toolbar.py:32]: Could have instantiated a generic page at the top and gotten a simpler version of this line:

```python
page = GenericPage(driver, url="")
...
page.title_contains("Internet for people")
```

Comment (sv-hyacoub) [reply tests/bookmarks_and_history/test_open_bookmarks_from_toolbar.py]: I went with this solution, looks cleaner but I got a couple of CI failures.

**Review by ben-c-at-moz — APPROVED** [review]
looks good, I'm going to merge since I also added some stuff that will help others' tests pass

**Pull request merged**